### DISCUSSION TITLE
GOM file errors

### DISCUSSION THREAD

**Discussion by camilofdz96** [Anura3D/Anura3D_OpenSource#142] [Q&A]
Dear community,

I'm reaching out to you to comment on two problems that I have seen related to the GOM file generation:

- I'm working on a 3D axisymmetric penetration model. It involves the simulation of a four-plate vane penetration into dry soil, so I was modeling the soil domain as a 90° wedge and only one of the fins to take advantage of the symmetry of the problem. Even though, the model seems to be very large since I'm having issues when generating the Anura3D files through the GiD problem type (Anura3D 2023 version). The GOM file is not written completely when the number of elements in the model is approximately higher than 150.000 and the GOM file size limit seems to be around 20.000 KB. 

- Another problem I have seen is that the GOM file is not saving the name of the contact surfaces that I define to compute reaction forces. As a result, just one Surf file is being generated and there is no distinction of the surfaces as shown below.

![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/148833059/1aa48083-acf1-4edd-8d10-d859b9bc96ad)

Thank you so much,

Camilo.

**Comment by theaerls17-star** [comment]
Hi Camilo,

I am working in something very similar to what you have and facing the same problem, Did you manage to solve it?
